utils/analyzermanager.go

```diff
@@ -70,6 +70,22 @@ func convertToApplicabilityStatus(status string) ApplicabilityStatus {
 	}
 }
 
+// for sorting
+var applicableMapToScore = map[string]int{
+	"Applicable":                4,
+	"NotApplicable":             3,
+	"ApplicabilityUndetermined": 2,
+	"NotCovered":                1,
+	"NotScanned":                0,
+}
```

Comment:
Did you make sure those values are correct? I think you are mixing the var names and the var value.
![image](https://github.com/jfrog/jfrog-cli-security/assets/49212512/4cb7001e-33a5-40fb-ae93-b1ed56c02603)
In addition, Not applicable should be the least priority, check out: `resultstable.go` -> `Severities` var

Reply:
@attiasas Good point! But checking it against the application sorting for applicable gives me the following order:
1. Applicable
2. Not Applicable  
3. Undetermined
4. Not Scanned
5. Not Covered